paparazzi/paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -593,14 +593,14 @@ class PaparazziPluginTest {
   @Test
   fun lifecycleOwnerUsages() {
     val fixtureRoot = File("src/test/projects/lifecycle-usages")
+    val snapshotsRunsDir = File(fixtureRoot, "build/reports/paparazzi/runs")
+    snapshotsRunsDir.deleteRecursively()
 
     gradleRunner
       .withArguments("testDebug", "--stacktrace")
       .runFixture(fixtureRoot) { build() }
 
-    val snapshotsDir = File(fixtureRoot, "build/reports/paparazzi/images")
-    val snapshots = snapshotsDir.listFiles()
-    assertThat(snapshots!!).hasLength(1)
+    assertThat(snapshotsRunsDir.listFiles()!!).hasLength(3)
```

Comment:
Thoughts on this? I wasn't able to assert the presence of 3 images because the 3 tests generate identical screenshots resulting in a single image saved on disk.

Reply:
ah right, this is why i've kept a loose convention of 1 test per test project.  remember setting different background colors to force 3 images?  while looking into the images dir is peeking into implementation details, peeking into the runs dir is kinda moreso, so it'd be nice to avoid that.

Reply:
I can do that

Reply:
done!